assets/js/lists/lists-added-list-item-dialog.liquid

```diff
@@ -78,7 +78,7 @@
 
                 <div class="col-xs-12">
                     <div class="text-right">
-                        <button ng-click="redirect('{{ 'account#/lists/myLists' | absolute_url }}')" class="btn" >
+                        <button ng-click="redirect('{{ 'account#/lists/' | absolute_url }}')" class="btn" >
```

Comment:
Should we leave slash in the end of the link?

Reply:
We can not remove slash because this breaks the AngularJS component routing. 
See the routing definition. 
https://github.com/VirtoCommerce/vc-demo-theme-b2b/blob/c52cc2e25ea91acd5e68261373e05af2214ce638/assets/js/account/account.js#L129